hardware/ariadne/bootloaders/ariadne/w5100.h

```diff
@@ -3,9 +3,9 @@
 
 
 /* Network settings */
-#define IP_ADDR     192,168,84,128
+#define IP_ADDR     192,168,1,128
 #define SUB_MASK    255,255,255,0
-#define GW_ADDR     192,168,84,1
+#define GW_ADDR     192,168,1,254
 #define MAC_ADDR    0xDE,0xAD,0xBE,0xEF,0xFE,0xED
```

Comment:
I think this change should be squashed into https://github.com/codebendercc/Ariadne-Bootloader/pull/27/commits/e4d898b4e7d4fd62f79633d52c6442ea6c2ff69a